README.md

```diff
@@ -2267,6 +2268,23 @@ Describes the status of a window of a door/liftgate etc.
 |`location`|Grid|True||
 |`state`|WindowState|True||
 
+### SeatStatus
+Describes the status of a parameter of seat.
+
+##### Parameters
+
+| Value |  Type | Mandatory | Description |
+| ---------- | ---------- |:-----------: |:-----------:|
+|`seatLocation`|SeatLocation|True||
+|`conditionActive`|Boolean|True||
+
+### SeatOccupancy
+##### Parameters
+
+| Value |  Type | Mandatory | Description |
+| ---------- | ---------- |:-----------: |:-----------:|
+|`seatsOccupied`|SeatStatus|False||Seat status array containing location and whether the seats are occupied.
```

Comment:
Please correct the table

Reply:
@atiwari9 Done, please check the latest updates